README.md

```diff
@@ -0,0 +1,38 @@
+tree-sitter-vimdoc
+==================
+
+This grammar intentionally support a subset of the vimdoc "spec"; predictable
+results are the primary goal, so that _output_ formats (e.g. HTML) are
+well-formed; the _input_ (vimdoc) is secondary. The first step should always be
+to try to fix the input (within reason) rather than insist on a grammar that
+handles vimdoc's endless quirks.
+
+Reference
+---------
+
+- quasi-spec: https://github.com/nanotee/vimdoc-notes
+
+Known issues
+------------
+
+`block_end` and `line_noeol` are intentionally _not_ alised to `block` and `line` until all they are
+a bit more stable. Currently their behavior is not perfect, so having the non-aliased forms in the
+test files helps notice their quirks.
```

Comment:
While this is a good idea to say it, having that for a parser version that might be shipped with a released version of neovim seems like a bad idea.

Why not just aliasing it ?

Reply:
Will alias them. Things have settled down now :)